Test code can't compile with MQTT_USE_WS off

/home/jonesmz/mqtt_cpp/test/underlying_timeout.cpp:304: error: ‘boost::beast’ has not been declared
         boost::beast::get_lowest_layer(socket),
                ^~~~~

```
    as::async_connect(
#if BOOST_VERSION >= 107000
        boost::beast::get_lowest_layer(socket),
#else  // BOOST_VERSION >= 107000
        socket.lowest_layer(),
#endif // BOOST_VERSION >= 107000
```


I think this line needs an additional check for if MQTT_USE_WS is defined.

--- a/test/underlying_timeout.cpp
+++ b/test/underlying_timeout.cpp
@@ -300,11 +300,7 @@ BOOST_AUTO_TEST_CASE( connect_tls_ashs ) {
     char buf;
 
     as::async_connect(
-#if BOOST_VERSION >= 107000
-        boost::beast::get_lowest_layer(socket),
-#else  // BOOST_VERSION >= 107000
         socket.lowest_layer(),
-#endif // BOOST_VERSION >= 107000
         it, end,
         [&]
         (boost::system::error_code const& ec, as::ip::tcp::resolver::iterator) {

